torch/lib/c10d/TCPStore.cpp

```diff
@@ -428,17 +455,26 @@ TCPStore::TCPStore(
       if (numWorkers.value_or(-1) >= 0 && waitWorkers) {
         waitForWorkers();
       }
+
+      // socket to handle requests from server
+      listenSocket_ = tcputil::connect(
+          tcpStoreAddr_, tcpStorePort_, /* wait= */ true, timeout_);
+      tcpStoreListener_ = std::make_unique<ListenThread>(listenSocket_);
```

Comment:
do we plan to use this listener to handle things other than watch callback? If not, does it make sense to call it `watchListner_` to be more specific?